### DISCUSSION TITLE
i am facing ssues on my project

### DISCUSSION THREAD

**Discussion by cselabaccess** [schreibfaul1/ESP32-audioI2S#830] [Q&A]
In file included from c:\Users\KARTHICK\Documents\Arduino\libraries\ESP32-audioI2S-master\src\aac_decoder\aac_decoder.cpp:15:0:
c:\Users\KARTHICK\Documents\Arduino\libraries\ESP32-audioI2S-master\src\aac_decoder\libfaad/neaacdec.h:40:20: fatal error: memory.h: No such file or directory
compilation terminated.
exit status 1

this is on my code  

#include "Arduino.h"
#include "WiFi.h"
#include "Audio.h" 
#define uart_en 15
#define RXp2 16      
#define TXp2 17
#define I2S_DOUT      25
#define I2S_BCLK      27
#define I2S_LRC       26

const char *ssid = "csergds";     
const char *password = "12345678";

Audio audio;

void setup()
{

  Serial.begin(115200);
  Serial2.begin(115200,SERIAL_8N1, RXp2, TXp2);

  WiFi.disconnect();
  WiFi.mode(WIFI_STA);
  WiFi.begin( ssid, password);

  while (WiFi.status() != WL_CONNECTED)
    delay(1500);

  audio.setPinout(I2S_BCLK, I2S_LRC, I2S_DOUT);
  audio.setVolume(100);
  audio.connecttospeech("Starting", "en"); // Google TTS
}

void loop()

{

  if (Serial2.available()) {
    String Answer = Serial2.readString();
  
    //-----
            // Split the answer into chunks and send each chunk to connecttospeech
        size_t chunkSize = 80;  // Define chunk size (adjust if necessary)
        for (size_t i = 0; i < Answer.length(); i += chunkSize) {
        
          String chunk = Answer.substring(i, (i + chunkSize));
          Serial.println(chunk);
          audio.connecttospeech(chunk.c_str(), "en");

          while(audio.isRunning()){
            audio.loop();
        

          }
    //--------
   
  }
 

}
audio.loop();

}

void audio_info(const char *info) {

}
Compilation error: exit status 1

**Comment by schreibfaul1** [comment]
memory.h is a system library. In the screenshot you can see where the file could be. 
![image](https://github.com/user-attachments/assets/f2fd661c-3e20-41dd-99f1-c2929027bf3f)
